contracts/BondCalculator.sol

```diff
@@ -0,0 +1,301 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {mulDiv} from "@prb/math/src/Common.sol";
+import {IVotingEscrow} from "./interfaces/IVotingEscrow.sol";
+import "./interfaces/IUniswapV2Pair.sol";
+
+interface ITokenomics {
+    /// @dev Gets number of new units that were donated in the last epoch.
+    /// @return Number of new units.
+    function getLastEpochNumNewUnits() external view returns (uint256);
+}
+
+/// @dev Only `owner` has a privilege, but the `sender` was provided.
+/// @param sender Sender address.
+/// @param owner Required sender address as an owner.
+error OwnerOnly(address sender, address owner);
+
+/// @dev Value overflow.
+/// @param provided Overflow value.
+/// @param max Maximum possible value.
+error Overflow(uint256 provided, uint256 max);
+
+/// @dev Provided zero address.
+error ZeroAddress();
+
+/// @dev Provided zero value.
+error ZeroValue();
+
+// Struct for discount factor params
+// The size of the struct is 96 + 64 + 64 = 224 (1 slot)
+struct DiscountParams {
+    // DAO set voting power limit for the bonding account
+    // This value is bound by the veOLAS total voting power
```

Comment:
Description is not very clear. what does the value affect?

Reply:
Added description in #182